### ISSUE TITLE
Back to version 0.4.10 after upgrade

### ISSUE THREAD

**Issue by Zzo911** [instabot-py/instabot.py#2403]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x ] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x ] Bug
- [ ] Feature Request
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] Are you running the code on your phone 
- [x ] If so, have you tried running the code on your local machine?

```
Running in docker on Synology NAS
```

#### Operating System? (include version)
- [ ] macOS
- [ ] Windows
- [ ] Rasbian Linux Raspberry PI (include flavour)
- [ x] Linux (include flavour)
- [ ] Android (include terminal)

```
Linux instabot 3.10.105 #24922 SMP Wed Jul 3 16:37:24 CST 2019 x86_64 GNU/Linux
```

#### Python Version Requirement
- [x ] I am using Python 3.6 or above

#### Exact Python Version?
```
3.7.4
```

#### Pip Version?
```
19.2.3
```

#### Instabot Version?
```
0.4.10, previously 0.5.9
```

#### How you use the instabot
- [ x] instabot-py -c instabot.config.yml --login "$IG_USER" --password "$IG_PASSWORD" --session-file "/app/session/$IG_USER.session" 
- [ ] instabot-py --login LOGIN --password PASSWORD
- [ ] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue

```
Instabot 0.5.9 request to upgrade to 0.5.10 but in fact it downgrade to 0.4.10
Logs from the command : python3 -m pip install instabot-py --upgrade --no-cache-dir

Upgrading instabot.py
Requirement already up-to-date: instabot-py in /usr/local/lib/python3.7/site-packages (0.4.10)
Requirement already satisfied, skipping upgrade: fake-useragent==0.1.11 in /usr/local/lib/python3.7/site-packages (from instabot-py) (0.1.11)
Requirement already satisfied, skipping upgrade: config42==0.3.2 in /usr/local/lib/python3.7/site-packages (from instabot-py) (0.3.2)
Requirement already satisfied, skipping upgrade: requests==2.22.0 in /usr/local/lib/python3.7/site-packages (from instabot-py) (2.22.0)
Requirement already satisfied, skipping upgrade: PyYAML==5.1.0 in /usr/local/lib/python3.7/site-packages (from instabot-py) (5.1)
Requirement already satisfied, skipping upgrade: sqlalchemy==1.3.4 in /usr/local/lib/python3.7/site-packages (from instabot-py) (1.3.4)
Requirement already satisfied, skipping upgrade: instaloader==4.2.8 in /usr/local/lib/python3.7/site-packages (from instabot-py) (4.2.8)
Requirement already satisfied, skipping upgrade: blessings==1.7.0 in /usr/local/lib/python3.7/site-packages (from instabot-py) (1.7)
Requirement already satisfied, skipping upgrade: Jinja2>=2.0 in /usr/local/lib/python3.7/site-packages (from config42==0.3.2->instabot-py) (2.10.1)
Requirement already satisfied, skipping upgrade: certifi>=2017.4.17 in /usr/local/lib/python3.7/site-packages (from requests==2.22.0->instabot-py) (2019.6.16)
Requirement already satisfied, skipping upgrade: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/site-packages (from requests==2.22.0->instabot-py) (1.25.3)
Requirement already satisfied, skipping upgrade: idna<2.9,>=2.5 in /usr/local/lib/python3.7/site-packages (from requests==2.22.0->instabot-py) (2.8)
Requirement already satisfied, skipping upgrade: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.7/site-packages (from requests==2.22.0->instabot-py) (3.0.4)
Requirement already satisfied, skipping upgrade: six in /usr/local/lib/python3.7/site-packages (from blessings==1.7.0->instabot-py) (1.12.0)
Requirement already satisfied, skipping upgrade: MarkupSafe>=0.23 in /usr/local/lib/python3.7/site-packages (from Jinja2>=2.0->config42==0.3.2->instabot-py) (1.1.1)
Starting instabot for username: renaud.perozzo
Session file: /app/session/me.session
INFO:instabot_py.config:Setting configuration from instabot.config.yml : OK
t;-10H Instabot Configurator 
[-4;2H
[-1;-34HCould not find user in settings. Would you like to add now? (yes/no): 
```

**Comment by soulpawa** [comment]
+1

**Comment by martygriffin** [comment]
+1

**Comment by mrallanodreman** [comment]
same here!
but is a official change?
something else change?

**Comment by nickgryg** [comment]
This has been fixed in the latest release 0.5.12. You can try it on your environment, just perform:
```
python3 -m pip install instabot-py --no-cache-dir --upgrade
```
